pkg/app/cockroachdb.go

```diff
@@ -0,0 +1,303 @@
+package app
+
+import (
+	"context"
+	"fmt"
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	"github.com/kanisterio/kanister/pkg/field"
+	"github.com/kanisterio/kanister/pkg/helm"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/log"
+	"github.com/kanisterio/kanister/pkg/poll"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const cReadyTimeout = 10 * time.Minute
+
+type CockroachDB struct {
+	name      string
+	namespace string
+	cacrt     string
+	tlscrt    string
+	tlskey    string
+	cli       kubernetes.Interface
+	chart     helm.ChartInfo
+}
+
+// NewCockroachDB Last tested working version "22.1.5"
+func NewCockroachDB(name string) App {
+	return &CockroachDB{
+		name: name,
+		chart: helm.ChartInfo{
+			Release:  appendRandString(name),
+			RepoName: helm.CockroachDBRepoName,
+			RepoURL:  helm.CockroachDBRepoURL,
+			Chart:    "cockroachdb",
+			Values: map[string]string{
+				"image.tag":        "v22.1.5",
+				"image.pullPolicy": "IfNotPresent",
+			},
+		},
+	}
+}
+
+func (c *CockroachDB) Init(context.Context) error {
+	cfg, err := kube.LoadConfig()
+	if err != nil {
+		return err
+	}
+	c.cli, err = kubernetes.NewForConfig(cfg)
+	return err
+}
+
+func (c *CockroachDB) Install(ctx context.Context, namespace string) error {
+	log.Info().Print("Installing cockroachdb cluster helm chart.", field.M{"app": c.name})
+	c.namespace = namespace
+
+	// Create the helm client
```

Comment:
let's remove these comment. The code itself is saying that new client is being created. We will have to remove the other comments as well, that are saying the same thing that obvious by looking at code.